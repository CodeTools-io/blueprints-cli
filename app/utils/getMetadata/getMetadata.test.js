const getMetadata = require('./getMetadata')

describe('getMetadata', () => {
  test('can get variances of blueprint instance name', () => {
    const result = getMetadata({
      blueprint: 'Component',
      blueprintInstance: 'Button',
    })

    expect(result).toMatchSnapshot()
  })
})
