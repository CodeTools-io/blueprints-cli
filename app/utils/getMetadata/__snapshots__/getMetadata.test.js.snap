// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getMetadata can get variances of blueprint instance name 1`] = `
Object {
  "blueprint": "Component",
  "blueprintInstance": "Button",
  "blueprintInstance_CamelCaseFormat": "button",
  "blueprintInstance_CamelCaseFormat_plural": "buttons",
  "blueprintInstance_CamelCaseFormat_singular": "button",
  "blueprintInstance_ClassFormat": "Button",
  "blueprintInstance_ClassFormat_plural": "Buttons",
  "blueprintInstance_ClassFormat_singular": "Button",
  "blueprintInstance_ConstantFormat": "BUTTON",
  "blueprintInstance_ConstantFormat_plural": "BUTTONS",
  "blueprintInstance_ConstantFormat_singular": "BUTTON",
  "blueprintInstance_DashedFormat": "button",
  "blueprintInstance_DashedFormat_plural": "buttons",
  "blueprintInstance_DashedFormat_singular": "button",
  "blueprintInstance_PascalCaseFormat": "Button",
  "blueprintInstance_PascalCaseFormat_plural": "Buttons",
  "blueprintInstance_PascalCaseFormat_singular": "Button",
  "blueprintInstance_SlugFormat": "button",
  "blueprintInstance_SlugFormat_plural": "buttons",
  "blueprintInstance_SlugFormat_singular": "button",
  "blueprintInstance_camelCaseFormat": "button",
  "blueprintInstance_camelCaseFormat_plural": "buttons",
  "blueprintInstance_camelCaseFormat_singular": "button",
  "blueprintInstance_dashed-format": "button",
  "blueprintInstance_dashed-format_plural": "buttons",
  "blueprintInstance_dashed-format_singular": "button",
  "blueprintInstance_pascalCaseFormat": "Button",
  "blueprintInstance_pascalCaseFormat_plural": "Buttons",
  "blueprintInstance_pascalCaseFormat_singular": "Button",
  "blueprintInstance_plural": "Buttons",
  "blueprintInstance_singular": "Button",
  "blueprintInstance_slug-format": "button",
  "blueprintInstance_slug-format_plural": "buttons",
  "blueprintInstance_slug-format_singular": "button",
  "blueprint_plural": "Components",
  "blueprint_singular": "Component",
}
`;
